saskatoon/member/admin.py

```diff
@@ -205,11 +207,49 @@ def deactivate_account(self, request, queryset):
     ]
 
 
+class PersonAdmin(admin.ModelAdmin):
+    list_display = (
+        '__str__',
+        'phone',
+        'email',
+        'street_number',
+        'street',
+        'neighborhood',
+        'newsletter_subscription',
+        'language',
+    )
+    list_filter = (
+        'neighborhood',
+        'city',
+        'language',
+        'newsletter_subscription',
+    )
+    search_fields = (
+        'first_name',
+        'family_name',
+        'phone',
+        'postal_code_cleaned',
+        'auth_user__email',
+    )
+
+    def get_queryset(self, request):
+        queryset = super().get_queryset(request)
+        queryset = queryset.annotate(
+            postal_code_cleaned=Replace(
+                'postal_code',
+                Value(" "),
+                Value("")
+            )
+        )
+        
+        return queryset
```

Comment:
nitty gritty: can we try avoiding committing extra whitespaces in the file? 
For example I see one on this empty line with my editor.
Really no big deal though!
Btw if you want we could go shorter with :
```
def get_queryset(self, request):
        queryset = super().get_queryset(request)
        return queryset.annotate(
            postal_code_cleaned=Replace(
                'postal_code',
                Value(" "),
                Value("")
            )
        )
```